components/ProgressView/src/MDCProgressView.h

```diff
@@ -86,4 +86,20 @@ IB_DESIGNABLE
          animated:(BOOL)animated
        completion:(void (^__nullable)(BOOL finished))completion;
 
+/**
+ Start the progress bar's indeterminate animation.
+ */
+- (void)startAnimating;
+
+/**
+ Stop the progress bar's indeterminate animation.
+ */
+- (void)stopAnimating;
+
+/**
+ Sets the progress bar to determinate and does a smooth animation between the indeterminate state
+ and the progress state using the current value for progress.
+ */
+- (void)moveToDeterminateSmoothly;
```

Comment:
Is this API required from functionality perspective? If not, we can probably turn `isIndeterminate` into a enum value to represent the progress view's style.